js/user.js

```diff
@@ -1,24 +1,21 @@
-(function () {
-    let sidebar = document.getElementById("content-left");
+(async function () {
+    let page = document.getElementById("main-inner");
     let loadCommonCourses = null;
 
-    if (sidebar) {
-        let button = createButton("splus-user-courses-in-common-btn", "Courses in Common");
-        let img = createElement("img", [], { src: chrome.runtime.getURL("imgs/plus-icon.png"), width: 18, style: { verticalAlign: "middle", paddingLeft: "4px" } });
-        button.prepend(img);
-        button.querySelector("input").style.paddingLeft = "4px";
-        button.style.cursor = "pointer";
-        button.addEventListener("click", () => {
-            if (loadCommonCourses == null) {
-                loadCommonCourses = getCoursesInCommon(document.location.href.match(/\/(\d+)\//)[1]);
-            }
-            openModal("user-courses-in-common-modal", { loadCommonCourses: loadCommonCourses, userId: document.location.href.match(/\/(\d+)\//)[1] });
-        });
+    const userID = document.location.href.match(/\/(\d+)\//)[1]

```

Comment:
I'd end each statement with a semicolon just to match the style of the rest of the code. 

Also in my experience I've run in to many strange bugs when dealing with semicolon auto-insertion.